megatron/arguments.py

```diff
@@ -924,6 +930,10 @@ def _add_data_args(parser):
                        help='Maximum sequence length to process.')
     group.add_argument('--variable_seq_lengths', action='store_true', default=None,
                        help='Enable variable sequence lengths.')
+    group.add_argument('--scalar_loss_mask', type=float, default=0.0,
+                       help=('Scalar to multiply the loss of the "masked out" '
+                             'tokens. Set to zero (default) to completely '
```

Comment:
can say `..of the masked out tokens (usually the assistant tokens)..`

Reply:
Added in [731021b](https://github.com/epfLLM/Megatron-LLM/pull/58/commits/731021b617883a3e1eb33ae928ddd9c300c00b3b)